src/merge.js

```diff
@@ -194,12 +187,10 @@ class Merge {
       } else {
         return this.pouch.put(doc)
       }
-    } else {
-      if (doc.tags == null) { doc.tags = [] }
-      if (doc.creationDate == null) { doc.creationDate = new Date() }
-      await this.ensureParentExistAsync(side, doc)
-      return this.pouch.put(doc)
     }
+    if (doc.tags == null) { doc.tags = [] }
```

Comment:
There are multiple places where we add default values when they are missing. All those could probably extracted to some metadata function(s). Not in the scope of this PR though.